dashboard/app/models/pd/regional_partner_contact.rb

```diff
@@ -25,6 +25,30 @@ class Pd::RegionalPartnerContact < ActiveRecord::Base
 
   before_save :update_regional_partner
 
+  after_create :send_regional_partner_contact_emails
+  def send_regional_partner_contact_emails
+    form = sanitize_and_trim_form_data_hash
+    matched_but_no_pms = false
+
+    if regional_partner_id
+      partner_id = RegionalPartner.find(regional_partner_id)
+      regional_partner_program_managers = RegionalPartnerProgramManager.where(regional_partner_id: partner_id)
+
+      if regional_partner_program_managers.empty?
+        matched_but_no_pms = true
+        Pd::RegionalPartnerContactMailer.unmatched(form, matched_but_no_pms).deliver_now
```

Comment:
@clareconstantine can I change this to `Pd::RegionalPartnerContactMailer.unmatched(form, matched_but_no_pms = true).deliver_now` and not need line 38?

Reply:
yes, although I don't usually see that syntax and it seems odd (slash the linter might complain) to set a variable and then never refer to it. You could change it to `Pd::RegionalPartnerContactMailer.unmatched(form, true).deliver_now`, but then no one knows what that `true` parameter means unless they look at the method definition. My first choice is the way you have it, but any of those work!

Reply:
Ah ok that makes sense- thanks! I'll leave it the way it is